### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 11

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#299] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110554](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/974d620a-4bd4-4afb-9e7e-5e9f3644ffc8)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/1114ea93-09c3-47bd-b890-568f3e97bad4)

**Comment by Elia-Belli** [comment]
### Osservazione sull'unicità
Considero _**nilpotenti banali**_ tutti gli elementi della classe $[0]\_{60}$.
In $\mathbb{Z}\_{60}$ $x$ è **_nilpotente_** $\iff x^n\equiv 0(60)\iff x^n=60k\iff 60|x^n\iff 2^2\cdot 3\cdot 5|x^n$
Quindi tra i fattori di $x$ devono essere presenti $2,3,5$ affinchè sia divisibile, il più piccolo numero in $\mathbb{Z}\_{60}$ che soddisfa tale condizione è $x=2\cdot 3\cdot 5=30$, il successivo è ottenuto aggiungendo come fattore il più piccolo dei primi, ovvero $2$, ma è esattamente $60=2^2\cdot 3\cdot 5$ nilpotente banale. 
Allora abbiamo dimostrato che $[30]$ è l'_**unico nilpotente non banale**_,

**Comment by AglaiaNorza** [comment]
Soluzione ad opera della tutor [Elisa Scandiuzzi](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md), lezione del 30 ottobre 2024 (`appunti_30_10_22.pdf`):

![1](https://github.com/user-attachments/assets/eb2770aa-7049-4942-bcb2-ae3bc3f4941c)
![2](https://github.com/user-attachments/assets/cf046ca2-3242-4532-869b-0e78ff76f725)
